addons/sale_project/models/sale_order_line.py

```diff
@@ -235,19 +235,30 @@ def _timesheet_create_task_prepare_values(self, project):
             allocated_hours = self._convert_qty_company_hours(self.company_id)
         sale_line_name_parts = self.name.split('\n')
         title = sale_line_name_parts[0] or self.product_id.name
-        description = '<br/>'.join(sale_line_name_parts[1:])
-        return {
+        res = {
             'name': title if project.sale_line_id else '%s - %s' % (self.order_id.name or '', title),
             'analytic_account_id': project.analytic_account_id.id,
             'allocated_hours': allocated_hours,
             'partner_id': self.order_id.partner_id.id,
-            'description': description,
             'project_id': project.id,
             'sale_line_id': self.id,
             'sale_order_id': self.order_id.id,
             'company_id': project.company_id.id,
             'user_ids': False,  # force non assigned task, as created as sudo()
         }
+        if (
+           'calendar_event_id' in self
```

Comment:
Same here, actually

Reply:
Thanks for the review! 
`calendar_event_id` is only defined if `website_appointment_sale` module is installed, so I believe we cannot removing the check here.

Reply:
If it is defined in `website_appointment_sale`, it shouldn't be referred to here?
Instead, the method should be overridden in that module to update "res".

Reply:
thanks for the review! I moved these lines to the appropriate bridge module introduced in this [PR](https://github.com/odoo/enterprise/pull/64981).